cells/std/nixago/just.cue

```diff
@@ -0,0 +1,29 @@
+import "strings"
+import "text/template"
+
+#Config: {
+	head: string | *""
+	tasks: [string]: [...string]
+}
+
+data: #Config
+
+_final: {
+    head: data.head
+    tasks: {
+        for task, steps in data.tasks {
+            "\(task)": strings.Join(steps, "\n    ")
+        }
+    }
+}
+
+tmpl:
+"""
```

Comment:
```suggestion
"""
# Autogenerated! DO NOT EDIT!
# This justfile is autogenerated via https://divnix.github.io/std/reference/std/nixago/just.html
# So that you can run the fundamental repository tasks with your favorit task runner.
# No need to install Nix, for now.
```

Reply:
https://divnix.github.io/std/reference/std/nixago/just.html would have to be made a thing, as well. :smile: